pyproject.toml

```diff
@@ -59,6 +59,7 @@ click = "^8.1"
 Mako = "^1.2"
 ni-python-styleguide = "^0.4"
 sphinx_rtd_theme = "1.0"
+poetry-grpc-plugin = "^0.1.0"
```

Comment:
We should be more careful about gRPC/protobuf versioning. Please add grpcio, grpcio-tools, and protobuf so that we can control their versions.

For measurementlink-python, we are synchronizing the main package's oldest compatible grpcio with the version of grpcio-tools used for codegen: https://github.com/ni/measurementlink-python/blob/main/pyproject.toml
However, the nidaqmx package won't depend on grpcio, so we will have to rely on runtime errors to notify users about mismatches.

Reply:
Update: let's start using the "extras" feature to capture the run-time dependencies.
https://python-poetry.org/docs/pyproject/#extras

That way, clients can explicitly enable gRPC support: 
https://github.com/ni/measurementlink-python/issues/239

```
grpcio = { version = "^1.49.0", optional = true }
protobuf = { version = "^4.21", optional = true }

[tool.poetry.extras]
grpc = ["grpcio", "protobuf"]
```